internal/vmpool/cloudaws/pool.go

```diff
@@ -35,9 +35,10 @@ type awsPool struct {
 	privateKey    string
 	iamProfileArn string
 
-	region  string
-	os      string
-	rootDir string
+	region           string
+	AvailabilityZone string
```

Comment:
maybe change this to a private field and create a getter method for it to match the implementation of the other fields.

Reply:
made the fix